docs/api/specs/browserResidentDesktopAgents.md

```diff
@@ -0,0 +1,253 @@
+---
+id: browserDesktopAgents
+sidebar_label: Browser Desktop Agents
+title: Browser-Resident Desktop Agents (next)
+---
+
+:::info _[@experimental](../fdc3-compliance#experimental-features)_
+
+Browser Resident Desktop Agents are an experimental feature added to FDC3 in 2.2. Limited aspects of their design may change in future versions and they are exempted from the FDC3 Standard's normal versioning and deprecation polices in order to facilitate any necessary change.
+
+:::
+
+This document specifies the required behavior for Browser-Resident Desktop Agents (DA). Such agents allow FDC3 applications running directly in a browser to participate in FDC3 interop by way of a `getAgent()` function that is provided by the [`@finos.fdc3` NPM module](https://www.npmjs.com/package/@finos/fdc3) and a standardized communication protocol. This approach is in contrast to "Preload DAs" which run on technology that allows the FDC3 interface to be injected (such as Electron, WebView2 or a browser-extension based implementation).
```

Comment:
I believe "`@finos.fdc3` NPM module" should read "`@finos/fdc3` NPM module"

Reply:
Fixed in 8c2cdfebfa99d0ac87ba9505f5b56af16e29b53a